scripts/wdl/docker_build_with_github_auth.wdl

```diff
@@ -23,7 +24,12 @@ task build_and_push {
         git clone ~{repo_url} .
         git checkout ~{commit_hash}
         echo -n "" >> .gcloudignore
-        gcloud builds submit --tag ~{registry_host}/~{software_name}:~{commit_hash} --timeout=24h
+        if [ -z ~{machine_type} ]
+        then
+            gcloud builds submit --tag ~{registry_host}/~{software_name}:~{commit_hash} --timeout=24h
```

Comment:
Instead of adding this switch here, why not add a default option?

Reply:
A default option for machine_type?  It seems like you can only get the default machine by not using the --machine-type flag, so I think I need the switch.